app/models/ability.rb

```diff
@@ -0,0 +1,16 @@
+class Ability
+  include CanCan::Ability
+  def initialize(user)
+    user ||= User.new 
+    if user.role == "manager"
+      can :manage, :all
+    elsif user.role == "member"
+      can :read, Project
+      can [:read, :update], Task
+      can [:read, :update], Comment, user_id: user.id
```

Comment:
member should be able to read all comments

Reply:
this is working as we expect. a member can read the other members comments. and member can edit his own comment.